packages/api/src/server.ts

```diff
@@ -102,12 +102,12 @@ export const createApp = (): {
     const apiLimiter = rateLimit({
       windowMs: 60 * 1000, // 1 minute
       max: async (req) => {
```

Comment:
@jacksonh probably we could remove `async` here too

Reply:
👍